gp-includes/things/translation.php

```diff
@@ -131,7 +131,7 @@ function for_translation( $project, $translation_set, $page, $filters = array(),
 			$where[] = $wpdb->prepare( 't.user_id = %d', ($user && $user->ID) ? $user->ID : -1 );
 		}
 
-		if ( ! current_user_can( 'manage_options' ) ) {
+		if ( ! GP::$permission->current_user_can( 'admin' ) ) {
```

Comment:
This was previously `->can( 'write', 'project', $project->id )` see https://glotpress.trac.wordpress.org/changeset/382.

(Does somebody know why this line even exists?)